src/components/ItaliaTheme/View/ServizioView/ServizioCostiVincoli.jsx

```diff
@@ -21,19 +21,29 @@ const messages = defineMessages({
 
 const ServizioCostiVincoli = ({ content }) => {
   const intl = useIntl();
+  const sectionTitleToRender = () => {
+    if (richTextHasContent(content.costi))
+      return intl.formatMessage(messages.costi_e_vincoli_header);
+    else if (
+      !richTextHasContent(content.costi) &&
+      richTextHasContent(content.vincoli)
+    )
+      return intl.formatMessage(messages.vincoli);
+  };
```

Comment:
Così facendo stai passando una funzione come `title` al `RichTextSection` sotto.. farei così:
```suggestion
  const sectionTitleToRender = (() => {
    if (richTextHasContent(content.costi))
      return intl.formatMessage(messages.costi_e_vincoli_header);
    else if (
      !richTextHasContent(content.costi) &&
      richTextHasContent(content.vincoli)
    )
      return intl.formatMessage(messages.vincoli);
  })();
```